typings/headroom.d.ts

```diff
@@ -0,0 +1,50 @@
+
+interface Tolerance {
```

Comment:
Can you use https://www.npmjs.com/package/@types/headroom instead? If it is outdated you can try to update the third party types.

Reply:
done